### PR TITLE
Changes to _script, LDA, yml

### PR DESCRIPTION
**Pull request by psalveson**
Added build_all_linux bash script to allow use on systems running bash shell (linux). It replicates the function of build_all.bat.

Added a line to _script/build_all.r to prevent errors by rmarkdown functions. Renames index.rmd to index.Rmd.

Added LDA.html to the _site.yml. Note: There appears to be two files named LDA.rmd. One in lesson 10, and one in lesson 9. Need to resolve?

Changed the name of a chunk in lesson 9 from import1 to import1abc. There were duplicate chunk names which prevented render_site() from finishing. As a note: all R chunk names across ALL rendered files have to have either no name, or a unique name (unique across all rendered files).

### CODE REVIEW

File: _web/LDA.rmd [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+	stringsAsFactors = TRUE)
+bugs
+```
+
+```{r}
+bugs.lda = lda(formula = bug ~ w.j1 + w.j2 + w.ae, method = "moment",
+         data = bugs, 
+         prior = c(1,1)/2)
+		 
+predict(bugs.lda, data.frame(w.j1 = c(194), w.j2 = c(124), w.ae = c(49)))$posterior
```
Comment (jbwalker26) [inline _web/LDA.rmd:36]: For all the constants, we may want to leave a comment or something to explain where they are coming from.

File: _web/_site.yml [hunk 2]
```diff
@@ -27,5 +27,7 @@ navbar:
           href: manova2.html
         - text: "Split-Plot ANOVA"
           href: split-plot.html
+        - text: "LDA"
+          href: LDA.html
     - text: "About"
       href: about.html
```
Comment (jbwalker26) [inline _web/_site.yml:33]: It is generally bad practice to not have a newline at the end of the file.

File: _web/_site/about.html [hunk 3]
```diff
@@ -265,6 +261,15 @@
 
 
 
+</head>
+
+<body>
+
+
+<div class="container-fluid main-container">
```
Comment (jbwalker26) [inline _web/_site/about.html:269]: There seems to be tons of newline spacing; unless this file is automatically generated by a program, we might want to trim the general spacing in the html files.

**Review by jbwalker26 — COMMENTED** [review]
Here is some feedback.

File: _web/_site.yml (hunk 2 above)
Comment (jbwalker26) [reply _web/_site.yml:33]: I might have placed the comment to the wrong file.

File: _web/LDA.rmd (hunk 1 above)
Comment (psalveson) [reply _web/LDA.rmd:36]: I recommend you open an issue. I'm not sure what this has to do with this pull request.

File: _web/_site.yml (hunk 2 above)
Comment (psalveson) [reply _web/_site.yml:33]: I'm not sure what you mean. What comment? What wrong file?

File: _web/_site/about.html (hunk 3 above)
Comment (psalveson) [reply _web/_site/about.html:269]: I'd open an issue.

File: _web/LDA.rmd (hunk 1 above)
Comment (jbwalker26) [reply _web/LDA.rmd:36]: I used to my company's convention of addressing all code concerns with new code in the new PR.  But I see your point for sure.

I opened an issue for it.

File: _web/_site.yml (hunk 2 above)
Comment (jbwalker26) [reply _web/_site.yml:33]: In the GitHub file compare, there is always a complainant on any file not ending with a newline.  But we can address newline issues in a different PR.

I guess I did post (what I meant by comment) to the right line of code with this issue (the file is right).

**Comment by cottieda** [comment]
Thanks for adding Linux support.  I don't have a Linux box at home available to test this, but I've confirmed this doesn't break the windows version.

File: _web/LDA.rmd (hunk 1 above)
Comment (cottieda) [reply _web/LDA.rmd:36]: I'd definitely recommend opening an issue or a pull request of your own in this case.  Peter is generally not going to care about style issues with code he didn't write or modify.

**Pull request merged**